lms/djangoapps/courseware/views/views.py

```diff
@@ -275,6 +275,7 @@ def courses(request):
     """
     courses_list = []
     course_discovery_meanings = getattr(settings, 'COURSE_DISCOVERY_MEANINGS', {})
+    set_default_filter = settings.FEATURES.get('ENABLE_COURSE_DISCOVERY_DEFAULT_LANGUAGE_FILTER')
```

Comment:
Is there an implementation reason why you chose to add a new `settings.FEATURES` flag instead of one of the toggles available from [edx-toggles](https://github.com/openedx/edx-toggles) like [SettingToggle](https://github.com/openedx/edx-toggles/blob/master/edx_toggles/toggles/internal/setting_toggle.py#L11) or [WaffleFlag](https://github.com/openedx/edx-toggles/blob/master/edx_toggles/toggles/internal/waffle/flag.py#L16)?

It's my understanding from [Feature Flags and Settings on edx-platform](https://openedx.atlassian.net/wiki/spaces/AC/pages/40862688/Feature+Flags+and+Settings+on+edx-platform) and [OEP-17](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0017-bp-feature-toggles.html) that we should be using these edx-toggles now, and that FEATURES flags are deprecated.

Reply:
@pomegranited No specific reason for using feature flags, except for maybe the relationship with `ENABLE_COURSE_DISCOVERY` feature flag. After reading the the OEP, I think we can using `WaffleSwitch` here.